src/libcore/slice/mod.rs

```diff
@@ -2494,7 +2494,7 @@ impl<'a, T> ExactSizeIterator for ExactChunks<'a, T> {
     }
 }
 
-#[unstable(feature = "fused", issue = "35602")]
+#[stable(feature = "fused", since = "1.25.0")]
```

Comment:
`ExactChunks` is still unstable so this should be `#[unstable(feature = "exact_chunks", issue = "47115")]`.